composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "league/html-to-markdown",
+    "name": "spresnac/html-to-markdown",
```

Comment:
We'll want to keep the package name as-is.  Would you mind reverting this change?